operations/enterprises.md

```diff
@@ -698,6 +698,98 @@ Adds a new task to the enterprise, optionally to a specified department.
 | --- | --- | --- | --- |
 | `TaskId` | string | required | Unique identifier of added task. |
 
+## Get all tasks
```

Comment:
Usually we put getters above add or update. Looking at it now, it is not that consistent. Consider having some system in it, either by entities or by action types.